lib/pull-request/merge-back.ts

```diff
@@ -37,7 +37,13 @@ export interface MergeBackStage {
   readonly after: string;
 }
 
-export interface AutoMergeBackOptions extends pr.AutoPullRequestOptions {
+/**
+ * AutoMergeBackOptions and AutoMergeBackProps both share all of these options,
+ * but AutoMergeBackProps can't extend AutoMergeBackOptions, as the 'stage' prop of AutoMergeBackOptions
+ * is only applicable to the AutoMergeBackOptions interface. This interface instead captures all of the
```

Comment:
> as the 'stage' prop of AutoMergeBackOptions is only applicable to the AutoMergeBackOptions interface

I don't understand this statement. Where is the `stage` property needed and not needed?

Reply:
Reworded.